test/utility/localoperator.jl

```diff
@@ -86,3 +87,48 @@ end
     tr_after = tr(last(only(H_shifted.terms)))
     @test abs(tr_before - tr_after) / abs(tr_before) < 1.0e-12
 end
+
+unitcells = [(1, 1), (2, 3), (3, 3), (4, 3)]
+@testset "Site rotations on $uc unitcell" for uc in unitcells
+    unrotated_inds = collect(CartesianIndices(uc))
+    # use reverse(uc) to account for transposing when using rotl90, rotr90 on non-square unit cells
+    rr_rotated_inds = rotr90(siterotr90.(collect(CartesianIndices(reverse(uc))), Ref(reverse(uc))))
+    ll_rotated_inds = rotr90(siterotr90.(collect(CartesianIndices(reverse(uc))), Ref(reverse(uc))))
```

Comment:
I think these are the same, did you mean to:

```suggestion
    rr_rotated_inds = rotr90(siterotl90.(collect(CartesianIndices(reverse(uc))), Ref(reverse(uc))))
    ll_rotated_inds = rotl90(siterotr90.(collect(CartesianIndices(reverse(uc))), Ref(reverse(uc))))
```

Reply:
Ah I see, passive vs active transformations are confusing...